hotshot-query-service/migrations/sqlite/V600__leaf2_remove_view.sql

```diff
@@ -0,0 +1,17 @@
+-- SQLite doesn't let you drop PK columns or restructure tables, so we make a temporary table
+-- and copy the data from the old leaf2 table.
+
+CREATE TABLE leaf2_new
```

Comment:
I don't like creating a new table here. Why don't we drop the existing primary key, make height the primary key and then remove view column?